src/modules/amp-caching/index.ts

```diff
@@ -1,3 +1,29 @@
+// @ts-ignore
+import router from 'workbox-routing';
+// @ts-ignore
+import { CacheFirst, StaleWhileRevalidate } from 'workbox-strategies';
+
 export function ampAssetsCaching() {
+  const CACHE_OPTIONS = {
```

Comment:
Since this is static, does it make sense to hoist outside of the function?

Reply:
done